### PR TITLE
doc: Add more details regarding the expected DNS configuration

### PR DESCRIPTION
**Pull request by cfergeau**
This commit tries to provide a more extensive description of the DNS
setup crc is trying to achieve.

This is related to https://github.com/code-ready/crc/issues/1093

### CODE REVIEW

File: docs/source/getting_started/content/ref_dns-configuration.adoc [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+
+The OpenShift cluster managed by {prod} uses 2 DNS domain names, `crc.testing` and `apps-crc.testing`.
+`crc.testing` is used for core OpenShift services.
+`apps-crc.testing` is used when accessing OpenShift applications deployed on the cluster.
+
+For example, the OpenShift API server will be exposed as `api.crc.testing` while the OpenShift console is accessed through `console-openshift-console.apps-crc.testing`
+These DNS domains are served by a `dnsmasq` DNS container running inside the {prod} virtual machine.
+
+Running [command]`{bin} setup` will adjust your system DNS configuration so that it can resolve these domains.
+When [command] `{bin} start` is run, additional checks will be done to verify DNS is properly configured.
```
Comment (praveenkumar) [inline docs/source/getting_started/content/ref_dns-configuration.adoc]: `When [command] `{bin} start`  runs,` ?

Comment (cfergeau) [reply docs/source/getting_started/content/ref_dns-configuration.adoc]: I don't know which form is better, I'll wait for kevin's comment before updating the PR.

File: docs/source/getting_started/content/ref_dns-configuration.adoc [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
 = DNS configuration details
 
+[id="dns-configuration-general{context}"]
+== General DNS setup
+
+The OpenShift cluster managed by {prod} uses 2 DNS domain names, `crc.testing` and `apps-crc.testing`.
+`crc.testing` is used for core OpenShift services.
+`apps-crc.testing` is used when accessing OpenShift applications deployed on the cluster.
+
+For example, the OpenShift API server will be exposed as `api.crc.testing` while the OpenShift console is accessed through `console-openshift-console.apps-crc.testing`
```
Comment (robin-owen) [inline docs/source/getting_started/content/ref_dns-configuration.adoc]: There's no period at the end of this sentence. :)

File: docs/source/getting_started/content/ref_dns-configuration.adoc (hunk 1 above)
Comment (robin-owen) [reply docs/source/getting_started/content/ref_dns-configuration.adoc]: I'd reverse the clauses and use active voice:

```adoc
Additional checks are done to verify DNS is properly configured when running [command]`{bin} start`.
```

File: docs/source/getting_started/content/ref_dns-configuration.adoc [hunk 3]
```diff
@@ -1,6 +1,19 @@
 [id="dns-configuration_{context}"]
 = DNS configuration details
 
+[id="dns-configuration-general{context}"]
+== General DNS setup
+
+The OpenShift cluster managed by {prod} uses 2 DNS domain names, `crc.testing` and `apps-crc.testing`.
+`crc.testing` is used for core OpenShift services.
+`apps-crc.testing` is used when accessing OpenShift applications deployed on the cluster.
```
Comment (robin-owen) [inline docs/source/getting_started/content/ref_dns-configuration.adoc]:
```adoc
The `crc.testing` domain is for core OpenShift services.
```

```adoc
The `apps-crc.testing` domain is for accessing OpenShift applications deployed on the cluster.
```

File: docs/source/getting_started/content/ref_dns-configuration.adoc [hunk 4]
```diff
@@ -1,6 +1,19 @@
...
+The OpenShift cluster managed by {prod} uses 2 DNS domain names, `crc.testing` and `apps-crc.testing`.
+The `crc.testing` domain is for core OpenShift services.
+The `apps-crc.testing` domain is for accessing OpenShift applications deployed on the cluster.
+
+For example, the OpenShift API server will be exposed as `api.crc.testing` while the OpenShift console is accessed through `console-openshift-console.apps-crc.testing`.
+These DNS domains are served by a `dnsmasq` DNS container running inside the {prod} virtual machine.
+
+Running [command]`{bin} setup` will adjust your system DNS configuration so that it can resolve these domains.
+Additional checks are done to verify DNS is properly configured when running [command]`{bin} start`.
+
```
Comment (praveenkumar) [inline docs/source/getting_started/content/ref_dns-configuration.adoc]: :+1:

**Pull request merged**